app/src/test/kotlin/com/intive/picover/parties/repository/PartiesRepositoryTest.kt

```diff
@@ -5,28 +5,41 @@ import com.google.firebase.database.DatabaseReference
 import com.google.firebase.database.FirebaseDatabase
 import com.google.firebase.database.Query
 import com.google.firebase.database.snapshots
+import com.intive.picover.common.coroutines.CoroutineTestExtension
 import com.intive.picover.parties.model.PartyRemote
 import io.kotest.assertions.throwables.shouldThrowExactly
+import io.kotest.core.spec.IsolationMode
 import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.result.shouldBeFailure
+import io.kotest.matchers.result.shouldBeSuccess
 import io.kotest.matchers.shouldBe
+import io.mockk.coEvery
+import io.mockk.coVerify
 import io.mockk.every
 import io.mockk.mockk
 import io.mockk.mockkStatic
 import io.mockk.unmockkAll
 import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.tasks.await
 
 class PartiesRepositoryTest : ShouldSpec(
 	{
+		extension(CoroutineTestExtension())
+		isolationMode = IsolationMode.InstancePerTest
 
 		val databaseReference: DatabaseReference = mockk()
 		val firebaseDatabase: FirebaseDatabase = mockk {
 			every { getReference("Parties") } returns databaseReference
 		}
 		val tested = PartiesRepository(firebaseDatabase)
+		val newKey = "newKey"
```

Comment:
I don't think it's worth to store it as class field as it is only used in one test, same for other properties.